src/java.base/share/classes/java/lang/package-info.java

```diff
@@ -25,35 +25,42 @@
 
 /**
  * Provides classes that are fundamental to the design of the Java
- * programming language. The most important classes are {@code
- * Object}, which is the root of the class hierarchy, and {@code
+ * programming language. The most important classes are {@link
+ * Object}, which is the root of the class hierarchy, and {@link
  * Class}, instances of which represent classes at run time.
  *
  * <p>Frequently it is necessary to represent a value of primitive
- * type as if it were an object. The wrapper classes {@code Boolean},
- * {@code Character}, {@code Integer}, {@code Long}, {@code Float},
- * and {@code Double} serve this purpose.  An object of type {@code
- * Double}, for example, contains a field whose type is double,
+ * type as if it were an object.<a id="wrapperClasses"></a>
+ * The <dfn>{@index "wrapper classes"}</dfn> {@link Boolean},
+ * {@link Character}, {@link Short}, {@link Integer}, {@link Long}, {@link Float},
+ * and {@link Double} serve this purpose.  An object of type {@code
```

Comment:
Somehow, `Byte` is missing from that list. To be fair, it had been missing before this PR.

Reply:
Good catch. After Byte is added, all 8 wrapper classes are present and accounted for.